Polygon_mesh_processing/test/Polygon_mesh_processing/CMakeLists.txt

```diff
@@ -95,3 +95,4 @@ endif(EIGEN3_FOUND)
   create_single_source_cgal_program("test_does_bound_a_volume.cpp")
   create_single_source_cgal_program("test_pmp_clip.cpp")
   create_single_source_cgal_program("triangulate_hole_polyline_test.cpp")
+  create_single_source_cgal_program("area.cpp")
```

Comment:
That file `area.cpp` does not exist. The PMP line in the testsuite was yellow.

https://cgal.geometryfactory.com/CGAL/testsuite/results-4.10-Ic-167.shtml#Polygon_mesh_processing